lib/features/auth/view/screens/sign_up_screen.dart

```diff
@@ -187,7 +195,7 @@ class SignUpScreenState extends State<SignUpScreen> {
                   iconSize: Sizes.iconSize,
```

Comment:
in line 170, use something called AuthTextButton instead of the normal TextButton. You can see how is it used in the log in screen.